### PR TITLE
Implement add-key option

### LINKED ISSUE #513: near-cli should support add key
Currently near-cli only supports deleting access keys, but doesn't support adding access keys.

### PR DESCRIPTION
**Pull request by ilblackdragon**
Fixes #513 

Usage:

Adding full access key to `illia` account:
```
near add-key CaSbwUJy2WEKKHHYnd6F7ZXUVUJ3AAzLBbDmvqML3jBg --account-id illia
```

Adding function access key to `illia` account for contract `test`, method name `deploy`, allowance: `1` $NEAR:
```
./bin/near add-key CaSbwUJy2WEKKHHYnd6F7ZXUVUJ3AAzLBbDmvqML3jBg --account-id illia --contract-id test --method-names deploy --allowance 1
```

PS. Where are we suppose to update documentation for this? @mikedotexe @potatodepaulo @amgando

### CODE REVIEW

File: commands/add-key.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        .option('allowance', {
+            desc: 'Allowance in $NEAR for the key (default 0)',
+            type: 'string',
+            required: false,
+        }),
+    handler: exitOnError(addAccessKey)
+};
+
+async function addAccessKey(options) {
+    if (!options.accountId) {
```
Comment (vgrichina) [inline commands/add-key.js:34]: @ilblackdragon shouldn't this throw error?

Comment (vgrichina) [reply commands/add-key.js:34]: I'll change to throw error unless there are any objections.

Comment (vgrichina) [reply commands/add-key.js:34]: or maybe just change to `add-key <account-id> <access-key>` as both seem to be require arguments

Comment (ilblackdragon) [reply commands/add-key.js:34]: I just copied this from delete-key -> so prob should change there too.

**Pull request merged**